client/src/commands/wrapLinesWithComment.ts

```diff
@@ -0,0 +1,80 @@
+import { Range, commands, window } from "vscode";
+
+/**
+ * Registers a command to wrap the non-blank content of the selected line or multiple lines with SAS block comments (/* ... *\/).
+ * If the selection spans multiple lines, each line will be commented individually.
+ */
+export function registerAddCommentCommand() {
+  return commands.registerCommand("SAS.addBlockComment", async () => {
+    const editor = window.activeTextEditor;
+    if (!editor) {
+      window.showInformationMessage("No active editor");
+      return;
+    }
+    const { selections, document } = editor;
+    await editor.edit((editBuilder) => {
+      for (const selection of selections) {
+        const start = selection.start.line;
+        const end = selection.end.line;
+        for (let i = start; i <= end; i++) {
```

Comment:
There's a problem. If you click a line number on the gutter, it will select the whole line including the line ending. And pressing Ctrl+/ will add comment also to the next line. No such problem with VS Code built-in language.